packages/table/src/table.tsx

```diff
@@ -316,8 +344,16 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
 
         const selectedRegions = (props.selectedRegions == null) ? [] as IRegion[] : props.selectedRegions;
 
+        let focusedCellCoordinates: IFocusedCellCoordinates = null;
+        if (props.focusedCellCoordinates != null) {
+            focusedCellCoordinates = props.focusedCellCoordinates;
+        } else if (props.allowFocus) {
+            focusedCellCoordinates = { col: 0, row: 0 };
```

Comment:
Do we want to unfocus a cell? I can't think of a good usecase -- spreadsheets don't let you do this, they always have one.